app/client/components/delivery/records/deliveryRecordsApi.ts

```diff
@@ -85,6 +85,8 @@ export const createDeliveryRecordsFetcher = (
   isTestUser: boolean
 ) => () =>
   fetch(`/api/delivery-records/${subscriptionId}`, {
+    credentials: "include",
+    mode: "same-origin",
     headers: {
       [MDA_TEST_USER_HEADER]: `${isTestUser}`
     }
```

Comment:
notice these headers, they might well need adding to the other fetches, so they 'test-users' works correctly, see https://github.com/guardian/manage-frontend/wiki/test-users#test-users